src/RankedPairsCalculator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace PivotLibre\Tideman;
+
+class RankedPairsCalculator
+{
+
+    /**
+     * @return CandidateList in which the zeroth Candidate is the most preferred, the first Candidate is the second most
+     * preferred, and so on until the last Candidate who is the least preferred.
+     */
+    public function calculate(NBallot ...$nBallots) : CandidateList
```

Comment:
This is best place to start reviewing. It is also most probably the method that the Laravel app will use to get  the winner. Do you think this method is workable from the Laravel app's perspective?